crates/wallet/src/lib.rs

```diff
@@ -98,7 +100,7 @@ pub trait OdysseyWalletApi {
     ///
     /// [eip-7702]: https://eips.ethereum.org/EIPS/eip-7702
     /// [eip-1559]: https://eips.ethereum.org/EIPS/eip-1559
-    #[method(name = "sendTransaction")]
+    #[method(name = "odyssey_sendTransaction")]
```

Comment:
when we deploy this we should keep the `wallet_sendTransaction` as an alias, so that we don't break anything on redeployment immediately

Reply:
question: will `odyssey_*` namespace will be only used in wallet related context or might potentially used in other context as well 

later case, considering seperate `OdysseyApi` trait than `WalletApi`. former case just have ` aliases = ["odyssey_sendTransaction"]` would be better

Reply:
I would say for this task we can just focus on adding an alias for the namespace.

Reply:
sure